tests/test_base_resources.py

```diff
@@ -441,6 +442,28 @@ def test_ep(self, reqmock):
         assert tm[0].num == 1.0
         assert tm[1].num == 2.0
 
+    def test_ep_auto_retry_true(self, reqmock):
+        responses = [
+            {'status_code': 503, 'headers': {'Retry-After': '0.01'}},
```

Comment:
It's probably not a big deal that this sleeps for .01s, but I'd probably mock it like we do here: https://github.com/analyzere/analyzere-python/blob/080baaad3d7cd3a43894a9abb88f625fa959ab01/tests/test_requestor.py#L140. Then you could assert it was called.